### DISCUSSION TITLE
Query regarding artillery pro "Scale" license

### DISCUSSION THREAD

**Discussion by JayasimhaNagaraj** [artilleryio/artillery#1090] [Artillery]
Hi Team,

Had a query regarding artillery pro "Scale" license.
Could you please help us understand on the below points :

1) Run up to **10 tests at the same time** --> Is it like running multiple yaml files or running multiple tests from different users.

2) No limits on number of test runs, test minutes, virtual users, or virtual user concurrency. Up to **1,000 workers per test.** --> What does worker denote in this ?

**Comment by hassy** [comment]
hi @JayasimhaNagaraj 👋 

1. 10 tests at the same time - is 10 different instances of `artillery` running a test. So test 1 can be running against API no. 1, whilst test 2 (with a different YAML script) is running against API no. 2 etc. More parallel tests are often needed by larger teams working on many services, and running tests from their CICD pipelines.
2. When you run a test with Artillery Pro, you have the option to horizontally scale it out (which in effect multiplies the load level). Each worker is a load generating container on ECS/Fargate executing the test.

Hope that makes sense, let us know if you have any further questions!
